cmd/stack.go

```diff
@@ -73,6 +73,11 @@ func setupStackCommand() *cobraext.Command {
 				return cobraext.FlagParsingError(err, cobraext.StackVersionFlagName)
 			}
 
+			/*serverless, err := cmd.Flags().GetString(cobraext.ServerlessFlagName)
```

Comment:
@mrodm @jsoriano Any recommendations on the path to follow here. I'm looking at geoip_dir which is part of a profile. At the same time I don't know if making it part of profiles is overkill? 

As you see further down, it is a variable in the template loaded from the resources. How do I set the resource variable on elastic-stack up in the best way? https://github.com/elastic/elastic-package/pull/1231/files#diff-9fd733fc9d8f10a5c2dd55a062523d3dcb0655bc1db097648b3adabbd3460173R123

Reply:
Hey, thanks for adding this. I would add it as a profile setting, in the same way as `geoip_dir`. Maybe in the future we can add a flag to pass this kind of settings through the command line, so we can run commands like `elastic-package stack up -Dstack.serverless=true`.

Reply:
I assume this would mean it can only be used by someone that adds a custom profile to elastic-package? The part I'm worried with this is that the feature will be tricky to use. I want to make it as easy as possible for everyone to test serverless by just adding `--serverless` like other params.

Reply:
If my understand is correct, the path forward is that we have multiple profiles available by default meaning users of elastic-package are likely already familiar by then with profiles. With this in mind, what you suggest makes a lot of sense. So if I want to switch between setups, I just switch the profiles.

Reply:
We don't ship predefined profiles (apart of the default one), but this would be definitely an option once we have more options.